pkg/gator/runner.go

```diff
@@ -33,19 +33,32 @@ type Runner struct {
 	includeTrace bool
 }
 
-func NewRunner(filesystem fs.FS, newClient func(includeTrace bool) (Client, error), includeTrace bool) (*Runner, error) {
+func NewRunner(filesystem fs.FS, newClient func(includeTrace bool) (Client, error), opts ...RunnerOptions) (*Runner, error) {
 	s := runtime.NewScheme()
 	err := apis.AddToScheme(s)
 	if err != nil {
 		return nil, err
 	}
 
-	return &Runner{
-		filesystem:   filesystem,
-		newClient:    newClient,
-		scheme:       s,
-		includeTrace: includeTrace,
-	}, nil
+	r := &Runner{
+		filesystem: filesystem,
+		newClient:  newClient,
+		scheme:     s,
+	}
+
+	for _, opt := range opts {
+		opt(r)
+	}
+
+	return r, nil
+}
+
+type RunnerOptions func(*Runner)
+
+func WithIncludeTrace(includeTrace bool) RunnerOptions {
```

Comment:
nit: `IncludeTrace()`

`WithIncludeTrace()` is more associated with customization-after-construction, e.g. `NewRunner(<args>).WithIncludeTrace()`